mainflux-loadtests/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.2")
```

Comment:
You can also add the native packager. It will be useful once we want to "deploy" our load testing tool.